### PR TITLE
LPS-187506 | Test Fix | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-187506](https://issues.liferay.com/browse/LPS-187506)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 1]
```diff
@@ -470,24 +470,29 @@ definition {
...
 
-		Workflow.viewWorkflowDefinition(workflowDefinitionTitle = "Test title");
+		ProcessBuilderKaleoDesignerReact.editConnector(
+			connectorLabel = "START-END CONNECTOR",
+			connectorLabelEdit = "pt_Start-End Connector");
 
-		ProcessBuilderKaleoDesignerReact.viewTranslationLanguageStatus(
-			newTranslationOption = "fr_FR",
-			translationStatus = "success");
+		ProcessBuilderKaleoDesignerReact.editWorkflowTitle(title = "pt_Workflow Definition");
```
Comment (davisaints) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:496]: @Luizcarlosqueiroz  who did this refactoring, but had some problems doing a rebase and what he wanted was not merged. I'm just redoing.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase [hunk 2]
```diff
@@ -1581,7 +1586,7 @@ definition {
 
 		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
-		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(translationOptionSelected = "en-us");
+		ProcessBuilderKaleoDesignerReact.openTranslationLanguageDropdown();
```
Comment (davisaints) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase:1590]: The same here

**Comment by patriciajeanperez** [comment]
@Luizcarlosqueiroz any objections forwarding this one?

**Comment by Luizcarlosqueiroz** [comment]
No objections! We can move forward :smile:

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**